src/main/java/com/company/Main.java

```diff
@@ -8,6 +8,5 @@
 
     public static void main(String[] args) {
```

Comment:
wyniesc do osobnej plasy w core pakietu tj. do com.company.Application.java
albo com.company.app.Application.java - nie mieszac z kontrolerami

Reply:
w ogole to zmien pakeity na pl.washboard.hvac.gamelibrary czy cos takiego